shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/PrivacyPolicyForVisitors.kt

```diff
@@ -20,5 +24,7 @@ fun PrivacyPolicyForVisitors() {
         subheader = stringResource(Res.string.privacy_policy_for_visitors_version),
         loadText = { Res.readBytes("files/visitors-privacy-policy.md") },
         onBack = navController::popBackStack
-    )
+    ) {
+        Spacer(Modifier.height(24.dp))
+    }
```

Comment:
Maybe this should be the default value for this parameter? There could also be padding that's always added at the end of the content (as I see it, we add it in all 3 usages so far), and then the default for this parameter could be `{}`.

Reply:
I want to revisit bottom paddings later